engine/storage/datamotion/src/main/java/org/apache/cloudstack/storage/motion/StorageSystemDataMotionStrategy.java

```diff
@@ -1654,18 +1654,18 @@ private ModifyTargetsCommand getModifyTargetsCommand(long storagePoolId, String
         details.put(ModifyTargetsCommand.STORAGE_HOST, storagePool.getHostAddress());
         details.put(ModifyTargetsCommand.STORAGE_PORT, String.valueOf(storagePool.getPort()));
 
-        ModifyTargetsCommand modifyTargetsCommand = new ModifyTargetsCommand();
+        ModifyTargetsCommand cmd = new ModifyTargetsCommand();
```

Comment:
how unjavaish of you @mike-tutkowski ;)

Reply:
It seems to fit in better with the general pattern I have observed in the codebase (using cmd instead of a longer name). :)